packages/api/schema/query.js

```diff
@@ -303,6 +304,10 @@ export default [
       Get a specific work unit by ID
       """
       work(workId: ID!): Work
+      """
+      Get List of currently registered worker plugins
+      """
+      activeWorkTypes: [WorkType]!
```

Comment:
[WorkType!]!

Reply:
I made it like that in the case where there is no worktype so user should expect empty array or array of worktype.

Reply:
I know it unlikely there will not be any work but its the right logic.

Reply:
[WorkType]! = [null], I don't think you want to return an array of null